content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -447,7 +447,10 @@ either be a snapshot file from a previous backup operation, or from a remaining
    ```
 
    If `<data-dir-location>` is the same folder as before, delete it and stop the etcd process before restoring the cluster. 
+
```

Comment:
nit: I don't think we need this empty line. The text won't follow if we add this.